### ISSUE TITLE
DataTable in full height

### ISSUE THREAD

**Issue by seszele64** [GibbsConsulting/django-plotly-dash#471]
Is there any way to render datatable in full height without it being truncated via scrolling? 

Example of how it looks like:
![image](https://github.com/GibbsConsulting/django-plotly-dash/assets/76787567/b3c87020-cd0f-4077-8e7b-57bdf55a9ce6)

I noticed it is due to "relative" position parameter in div enclosing the embeded iframe, however it is set by default and i dont know a way to change it.
![image](https://github.com/GibbsConsulting/django-plotly-dash/assets/76787567/bd6e7230-ee90-4932-9cfd-99db027ca5e7)

When you remove the relative, it reverts to full height table.
![image](https://github.com/GibbsConsulting/django-plotly-dash/assets/76787567/86be4887-a3a6-482a-b47f-aecacee0ebb0)

Hoping to see an answer, cya!

**Comment by delsim** [comment]
@seszele64 which template tag are you using to render the app? The `plotly_dash` tag at the moment does insert relative position; it would be straightforward to make this optional.

**Comment by seszele64** [comment]
Yes, indeed. In case anybody wants to find it and edit according to their needs, the directory is: `django_plotly_dash/templatetags/plotly_dash.py`

![image](https://github.com/GibbsConsulting/django-plotly-dash/assets/76787567/c22ba828-5674-4446-9e85-2a54cb671932)
